Dockerfile

```diff
@@ -24,12 +24,6 @@ RUN git clone https://github.com/multiversx/mx-chain-proxy-go.git --branch=${PRO
 RUN git clone https://github.com/multiversx/mx-chain-proxy-go.git --branch=${PROXY_DEVNET_TAG} --single-branch --depth=1 mx-chain-proxy-go-devnet
 RUN git clone https://github.com/multiversx/mx-chain-proxy-go.git --branch=${PROXY_MAINNET_TAG} --single-branch --depth=1 mx-chain-proxy-go-mainnet
 
-# Adjust node source code
-COPY "adjust_observer_src.py" /workspace/
```

Comment:
Not needed anymore, synchronization (as a full history node) works faster since Sirius.